Maple2.Server.Web/Controllers/WebController.cs

```diff
@@ -44,22 +44,25 @@ public class WebController : ControllerBase {
 
         byte[] fileBytes = packet.ReadBytes(packet.Available);
 
+        UgcResource? resource = null;
         if (ugcUid != 0) {
             using WebStorage.Request db = webStorage.Context();
-            UgcResource? resource = db.GetUgc(ugcUid);
+            resource = db.GetUgc(ugcUid);
             if (resource == null) {
                 return Results.NotFound($"{ugcUid} does not exist.");
             }
-            if (System.IO.File.Exists(resource.Path)) {
+            /*if (System.IO.File.Exists(resource.Path)) {
```

Comment:
is this intentionally commented?

Reply:
Ah I was having issues with it due to Item and Item Icon colliding (both get uploaded one after another). I pushed an update that should address this.